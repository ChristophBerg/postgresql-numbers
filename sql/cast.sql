SELECT 'vier'::zahl::bigint;
SELECT 'vier'::zahl + 5;
