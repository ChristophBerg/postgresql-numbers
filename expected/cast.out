SELECT 'vier'::zahl::bigint;
 int8 
------
    4
(1 row)

SELECT 'vier'::zahl + 5;
 ?column? 
----------
        9
(1 row)

